ols/src/prompts/prompt_generator.py

```diff
@@ -16,13 +16,13 @@
 from .prompts import CHAT_PROMPT
 
 PromptConfiguration = namedtuple(
-    "PromptConfiguration", "provider model rag_exists history_exists"
+    "PromptConfiguration", "provider model"
 )
 
 # TODO: this might be expanded in the future.
 prompt_configurations = {
-    PromptConfiguration(PROVIDER_BAM, GRANITE_13B_CHAT_V1, True, True): CHAT_PROMPT,
-    PromptConfiguration(PROVIDER_OPENAI, GPT35_TURBO, True, True): CHAT_PROMPT,
+    PromptConfiguration(PROVIDER_BAM, GRANITE_13B_CHAT_V1): CHAT_PROMPT,
+    PromptConfiguration(PROVIDER_OPENAI, GPT35_TURBO): CHAT_PROMPT,
```

Comment:
can you please expand this to our full set of "tier 1" provider/model pairs?

watsonx/granite13b-chatv2  
azure-openai/gpt35

also we shouldn't be using chat v1, should be chat v2.

see: 
https://github.com/openshift/lightspeed-service/blob/578a030ed1eecfbb193077dfb8db91e695265d38/tests/scripts/test-e2e-cluster.sh#L45-L54

Reply:
added